alacritty_terminal/src/config/bindings.rs

```diff
@@ -71,113 +71,158 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
     let mut bindings = bindings!(
         KeyBinding;
         Key::Paste; Action::Paste;
-        Key::Copy; Action::Copy;
+        Key::Copy;  Action::Copy;
```

Comment:
Worlds smallest nit-pick. But I think you've added an extra space here :stuck_out_tongue_winking_eye: